common/src/main/java/org/figuramc/figura/animation/Keyframe.java

```diff
@@ -83,26 +96,43 @@ private float parseStringData(String data, float delta) {
         return FiguraMod.popReturnProfiler(0f);
     }
 
+    @LuaWhitelist
+    @LuaMethodDoc("keyframe.ph")
```

Comment:
`@LuaMethodDoc` is not required for a function to be accessible in Lua, but if you are adding docs, please do it properly or remove useless "docs".

Reply:
From what i can tell thats required to have it show up in the docs command, they're all pretty self explanatory, so documentation shouldnt really be necessary imo

Reply:
"setPos" and "setRot" are also self-explanatory but still are documented, and it's better to have consistent documentation

Reply:
ugh fine ill add docs